00:00:00.002883 main     VirtualBox COM Server 7.0.4 r154605 win.amd64 (Nov 16 2022 15:33:13) release log
00:00:00.002885 main     Log opened 2023-01-04T15:11:00.752483300Z
00:00:00.002886 main     Build Type: release
00:00:00.002890 main     OS Product: Windows 11
00:00:00.002891 main     OS Release: 10.0.22000
00:00:00.002891 main     OS Service Pack: 
00:00:00.212976 main     DMI Product Name: 82H8
00:00:00.228239 main     DMI Product Version: IdeaPad 3 15ITL6
00:00:00.228249 main     Firmware type: UEFI
00:00:00.228537 main     Secure Boot: VERR_PRIVILEGE_NOT_HELD
00:00:00.228556 main     Host RAM: 7991MB (7.8GB) total, 2230MB (2.1GB) available
00:00:00.228560 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.228561 main     Process ID: 95736
00:00:00.228561 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.328430          VirtualBox: object creation starts
00:00:00.328615          Home directory: 'C:\Users\harsh\.VirtualBox'
00:00:00.331953          Installed Drivers:
00:00:00.337579            C:\Windows\system32\DRIVERS\VBoxSup.sys (Version: 7.0.4.4605)
00:00:00.337969            C:\Windows\system32\DRIVERS\VBoxUSBMon.sys (Version: 7.0.4.4605)
00:00:00.338366            C:\Windows\system32\DRIVERS\VBoxNetAdp6.sys (Version: 7.0.4.4605)
00:00:00.338628            C:\Windows\system32\DRIVERS\VBoxNetLwf.sys (Version: 7.0.4.4605)
00:00:00.338887          Loading settings file "C:\Users\harsh\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:00.341497          usbLibGetDevices: Starting USB device enumeration
00:00:00.380896          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&3881863d&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 5
00:00:00.381022          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&3881863d&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
00:00:00.385317          usbLibGetDevices: Found 2 USB devices, 0 captured
00:00:00.385412          USB: Unknown USB device detected (idVendor: 0x0bda, idProduct: 0xc123)
00:00:00.385497          USB: Unknown USB device detected (idVendor: 0x174f, idProduct: 0x2459)
00:00:00.555267          HostDnsMonitor: initializing
00:00:00.561258          HostDnsMonitor: updating information
00:00:00.561282          HostDnsMonitor: old information
00:00:00.561283            no server entries
00:00:00.561283            no domain set
00:00:00.561283            no search string entries
00:00:00.561284          HostDnsMonitor: new information
00:00:00.561284            server 1: 192.168.1.1
00:00:00.561285            domain: home
00:00:00.561285            no search string entries
00:00:00.612140          VD: VDInit finished with VINF_SUCCESS
00:00:00.613505          Loading settings file "C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\ISE_14.7_VIRTUAL_MACHINE.vbox" with version "1.19-windows"
00:00:00.614428          VirtualBox: object created
00:00:00.717836          Launched VM: 2289808320 pid: 95424 (0x174c0) frontend: GUI/Qt name: ISE_14.7_VIRTUAL_MACHINE
00:00:09.748557          ERROR [COM]: aRC=VBOX_E_OBJECT_NOT_FOUND (0x80bb0001) aIID={70401eef-c8e9-466b-9660-45cb3e9979e4} aComponent={ExtPackManagerWrap} aText={No extension pack by the name 'Oracle VM VirtualBox Extension Pack' was found}, preserve=false aResultDetail=0
00:00:18.106647          ERROR [COM]: aRC=VBOX_E_INVALID_VM_STATE (0x80bb0002) aIID={300763af-5d6b-46e6-aa96-273eac15538a} aComponent={SessionMachine} aText={The machine is not mutable or saved (state is Restoring)}, preserve=false aResultDetail=0
00:00:18.665973          NetworkServiceRunning: started 'VBoxNetDHCP.exe', pid 0001436c
00:00:21.137947          ERROR [COM]: aRC=VBOX_E_OBJECT_NOT_FOUND (0x80bb0001) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Property 'CRYPT/KeyId' does not exist}, preserve=false aResultDetail=0
00:00:21.140004          Added USB filter (ID=1, type=4) for device 03FD:FFFFFFFF rev FFFFFFFF, c/s/p FFFFFFFF/FFFFFFFF/FFFFFFFF, Manufacturer=`<null>' Product=`<null>' Serial=`<null>'
00:00:21.140144          Added USB filter (ID=2, type=4) for device 0403:FFFFFFFF rev FFFFFFFF, c/s/p FFFFFFFF/FFFFFFFF/FFFFFFFF, Manufacturer=`<null>' Product=`<null>' Serial=`<null>'
00:00:21.140168          Added USB filter (ID=3, type=4) for device 1443:FFFFFFFF rev FFFFFFFF, c/s/p FFFFFFFF/FFFFFFFF/FFFFFFFF, Manufacturer=`<null>' Product=`<null>' Serial=`<null>'
00:00:33.129582          Saving settings file "C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\ISE_14.7_VIRTUAL_MACHINE.vbox" with version "1.19-windows"
00:00:33.136234          Finished saving settings file "C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\ISE_14.7_VIRTUAL_MACHINE.vbox"
02:23:29.313636 USBPROXY usbLibGetDevices: Starting USB device enumeration
02:23:30.707651 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&3881863d&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 5
02:23:30.708168 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&3881863d&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
02:23:30.989392 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
02:30:31.986917 USBPROXY usbLibGetDevices: Starting USB device enumeration
02:30:32.016378 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&3881863d&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 5
02:30:32.017081 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&3881863d&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
02:30:32.021787 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
03:42:07.607503          Removed USB filter ID=1
03:42:07.608069          Removed USB filter ID=2
03:42:07.608078          Removed USB filter ID=3
03:42:07.648960          Saving settings file "C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\ISE_14.7_VIRTUAL_MACHINE.vbox" with version "1.19-windows"
03:42:07.661466          Finished saving settings file "C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\ISE_14.7_VIRTUAL_MACHINE.vbox"
03:42:07.882688 SaveState Saving settings file "C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\ISE_14.7_VIRTUAL_MACHINE.vbox" with version "1.19-windows"
03:42:07.886882 SaveState Finished saving settings file "C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\ISE_14.7_VIRTUAL_MACHINE.vbox"
03:42:14.689805 main     VirtualBox: object deletion starts
03:42:14.689880 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\14.7_VM-disk001.vmdk' cannot be closed because it is still attached to 1 virtual machines}, preserve=false aResultDetail=0
03:42:14.692235 main     NetworkServiceRunning: killing VBoxNetDHCP.exe, pid 0001436c...
03:42:14.697987 main     HostDnsMonitor: shutting down ...
03:42:14.698071 main     HostDnsMonitor: shut down
03:42:14.700906 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={7682d5eb-f00e-44f1-8ca2-99d08b1cd607} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
03:42:14.701639 main     VirtualBox: object deleted
